platform/darwin/src/MGLRasterSource.h

```diff
@@ -1,18 +1,57 @@
 #import "MGLSource.h"
 #import "MGLTypes.h"
 
+#import <CoreGraphics/CoreGraphics.h>
+
 @class MGLTileSet;
 
 NS_ASSUME_NONNULL_BEGIN
 
+/**
+ A raster tile source.
+ 
+ @see <a href="https://www.mapbox.com/mapbox-gl-style-spec/#sources-raster">The
+ style specification.</a>
+ */
 @interface MGLRasterSource : MGLSource
 
+/**
+ A URL to a TileJSON resource. Supported protocols are `http:`, `https:`, and 
```

Comment:
We should link to the TileJSON specification, in case the developer wants to author their own.